setup.py

```diff
@@ -60,6 +60,8 @@ def get_readme():
     install_requires=[
         # Dependencies go here!
         'numpy>=1.8',
+        'matplotlib',
```

Comment:
I don't think we need the matplotlib dependency, do we? So could you please remove it?